### PR TITLE
Info2-005-01-03: Add solution

### PR DESCRIPTION
**Pull request by Patertuck**
Not sure if I understood the task for exercise 3 correctly but here is my solution. It sorts which is good 😅

### CODE REVIEW

**Comment by C0DECYCLE** [comment]
(Forgot the prefix in the second commit.)

File: Informatics II/005/Task 3/Patertuck Solution/Task3.c [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#include <stdio.h>
+
+void swap(int *A, int i, int j)
+{
+    int temp = A[i];
+    A[i] = A[j];
+    A[j] = temp;
+}
+
+int partition(int *A, int r, int l){
```
Comment (Twhining) [inline Informatics II/005/Task 3/Patertuck Solution/Task3.c:10]: it just triggers my autism that right is on the left side and left is on the right side but this is really nitpicky.

Comment (C0DECYCLE) [reply Informatics II/005/Task 3/Patertuck Solution/Task3.c:10]: @Twhining like the way you think :)

**Review by Twhining — APPROVED** [review]
LGTM

**Pull request merged**